plugins/crypto/ua_certificategroup_filestore.c

```diff
@@ -0,0 +1,717 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ *    Copyright 2022 (c) Mark Giraud, Fraunhofer IOSB
+ *    Copyright 2023 (c) Fraunhofer IOSB (Author: Kai Huebl)
+ *    Copyright 2024 (c) Fraunhofer IOSB (Author: Noel Graf)
+ */
+
+#include <open62541/util.h>
+#include <open62541/plugin/certificategroup_default.h>
+#include <open62541/plugin/log_stdout.h>
+
+#include "mp_printf.h"
+
+#ifdef UA_ENABLE_ENCRYPTION
+
+#ifdef __linux__ /* Linux only so far */
+
+#include <stdio.h>
+#include <linux/limits.h>
+#include <libgen.h>
+#include <sys/stat.h>
+#include <sys/inotify.h>
+#include <unistd.h>
+#include <fcntl.h>
+#include <dirent.h>
+#include <bits/stdio_lim.h>
+
+#define EVENT_SIZE (sizeof(struct inotify_event))
+#define BUF_LEN (1024 * ( EVENT_SIZE + 16 ))
+
+struct FileCertStore;
+typedef struct FileCertStore FileCertStore;
+
+struct FileCertStore {
+    /* Memory cert store as a base */
+    UA_CertificateGroup *store;
+
+    int inotifyFd;
+
+    UA_String trustedCertFolder;
+    UA_String trustedCrlFolder;
+    UA_String issuerCertFolder;
+    UA_String issuerCrlFolder;
+    UA_String rejectedCertFolder;
```

Comment:
Persistence of rejected certificates is mainly a nice-to-have because it is not required by the spec. Nevertheless I just wanted to ask, if there are any limits applied on how many rejected certificates are kept?